modules/jfx.incubator.richtext/src/main/java/jfx/incubator/scene/control/richtext/CodeArea.java

```diff
@@ -417,16 +419,19 @@ public StyleableProperty<Number> getStyleableProperty(CodeArea n) {
      * @return plain text
      */
     public final String getText() {
-        // TODO or use save(DataFormat, Writer) ?
         StyledTextModel m = getModel();
-        StringBuilder sb = new StringBuilder(4096);
-        int sz = m.size();
-        for(int i=0; i<sz; i++) {
-            String s = m.getPlainText(i);
-            sb.append(s);
-            sb.append('\n');
+        if (m == null) {
+            return null;
```

Comment:
This didn't used to return `null` before this fix (it would throw NPE on a null model). Perhaps returning the empty string be better? Either way, it would be helpful to document whether or not this method can return null.